README.md

```diff
@@ -23,115 +22,117 @@ $ yarn add webpack-sentry-plugin --dev
 
-  ```js
-  var config = {
-    plugins: [
-      new SentryPlugin({
-        filenameTransform: function(filename) {
-          return 'a-filename-prefix-' + filename
-        }
-      })
-    ]
-  }
-  ```
+```js
+var config = {
+  plugins: [
+    new SentryPlugin({
+      filenameTransform: function(filename) {
+        return 'a-filename-prefix-' + filename
+      }
+    })
+  ]
+}
+```
   
-  - `releaseBody`: Object or function that returns the body that will be sent to Sentry. Defaults to sending the version.
-
-  ```js
-  var config = {
-    plugins: [
-      new SentryPlugin({
-        releaseBody: function(version) {
-          return { 
-            version,
-            refs: [
-               repository: 'project-repo',
-               commit: process.env.GIT_SHA
-            ]
-          }
+- `releaseBody`: Object or function that returns the body that will be sent to Sentry. Defaults to sending the version and projects (which is sufficient to create a basic new release in Sentry).
+  - The function is given two arguments: `version` and `projects`. `version` is the result of the `release` object (string or function output); `projects` is the `project` configuration parameter (converted to an array if a single string is provided).
+  - The most common use case for overriding this field is Sentry's [release commits](https://docs.sentry.io/learn/releases/#releases-are-better-with-commits) feature. To use this, define `releaseBody` per the example below (providing the most recent commit hash through whatever means works best for your build setup). See the Sentry documentation for more details and options.
+
+```js
+var config = {
+  plugins: [
+    new SentryPlugin({
+      releaseBody: function(version, projects) {
+        return { 
+          version,
+          projects,
+          refs: [
+             repository: 'project-repo',
+             commit: process.env.GIT_SHA
+          ]
         }
-      })
-    ]
-  }
-  ```
+      }
+    })
+  ]
+}
+```
 
-- `suppressErrors`: Display warnings instead of failing webpack build
+- `suppressErrors`: Display warnings for any errors instead of failing webpack build
 
-- `suppressConflictError`: Similar to `suppressErrors`, but only supresses release conflict errors - useful in case webpack compilation is done during deploy on multiple instances
+- `suppressConflictError`: Similar to `suppressErrors`, but only supresses release conflict errors - useful in case webpack compilation is done during deploy on multiple instances. (Release conflict errors are HTTP 409 errors thrown by Sentry when the same source map file is uploaded to the same release multiple times.)
 
-- `baseSentryURL`: URL of Sentry instance. Shouldn't need to set if using sentry.io, but useful if self hosting
+- `baseSentryURL`: URL of Sentry instance. Unnecessary if using sentry.io, but useful if self-hosting. Should be a fully qualified domain name with the suffix `/api/v0`, i.e. `https://mysentryinstance.com/api/v0`
```

Comment:
How about this:

> Fully qualified URL of Sentry instance. Defaults to `'https://sentry.io/api/0'` for `sentry.io`. If self-hosting, set this to the fully qualified domain name of your instance. eg. `https://mysentryinstance.com/api/v0`

I think showing the default value makes it a bit clearer how the value is used. Also it's an `eg` not an `ie`.